util/plhash.c

```diff
@@ -927,6 +927,24 @@ static unsigned int hash_default_strlen(const unsigned char *key,
     return hash;
 }
 
+static unsigned int hash_default_strcaselen(const unsigned char *key,
+                                            int len __attribute_unused__)
+{
+    unsigned hash = 0;
+    uint8_t k = *key;
+
+    while (k) {
+        /* Convert to lowercase */
+        if ((k > 64) && (k < 91)) {
```

Comment:
This will not work for the extended ASCII codes (128 - 255), for instance, ã <-> Ã. Maybe just call `tolower()` here.

Reply:
Sure, done.